src/main/osd/osd_elements.c

```diff
@@ -2107,6 +2109,66 @@ bool osdDrawNextActiveElement(displayPort_t *osdDisplayPort, timeUs_t currentTim
     return retval;
 }
 
+#ifdef USE_SPEC_PREARM_SCREEN
+void osdDrawSpec(displayPort_t *osdDisplayPort)
+{
+    if (!ARMING_FLAG(ARMED) && osdConfig()->osd_show_spec_prearm)
+    {
```

Comment:
Betaflight uses 1TBS (K&R) indent style
```suggestion
    if (!ARMING_FLAG(ARMED) && osdConfig()->osd_show_spec_prearm) {
```